packages/server/src/db/index.js

```diff
@@ -540,8 +540,120 @@ function grantsQuery(queryBuilder, filters, agencyId, orderingParams, pagination
     }
 }
 
+function formatSearchCriteriaToQueryFilters(criteria) {
```

Comment:
Honest question: Would it be better to convert the search calls to fit the serialized db representation rather than change the db representation to fit the search query? I'm trying to think about when input rules and fields change, which is likely on the client, but might be an afterthought for the notifications.